### DISCUSSION TITLE
Display Sandboxed tests on Problems tab

### DISCUSSION THREAD

**Discussion by alisevych** [UnitTestBot/UTBotJava#1270] [General]
**Description**

If an executions performs a restricted operation it is interrupted by Java Security Manager.
Those executions are presented as test methods and `disabled due to sandbox`.
They contain a stack trace of the restricted method call.
User can add a corresponding Permission or disable useSandbox functionality (by adding `useSandbox=false` in settings.properties).

**Expected behavior**

Would it be appropriate to display disabled tests as on Problems view with stack traces?
Because they are not errors.

It can be displayed like that:

![image](https://user-images.githubusercontent.com/37301492/199016433-db62c372-a8f0-47de-9c83-dbec18ce24d8.png)

User would be able to navigate the code that leads to the restricted operation.

**Environment**

IntelliJ IDEA 2021.2.4 - 2022.2.3 (all supported)

**Comment by alisevych** [comment]
@korifey @Markoutte @denis-fokin 
Please advise - would it be appropriate to display restricted cases on Problems view?

**Reply by korifey** [reply]
agree

**Comment by alisevych** [comment]
This functionality was implemented in 
- #1276
